modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/enricher/PasswordModelEnricher.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.introspection.enricher;
+
+import org.mule.extension.annotation.api.param.display.Password;
+import org.mule.extension.api.introspection.declaration.DescribingContext;
+import org.mule.extension.api.introspection.declaration.fluent.DeclarationDescriptor;
+import org.mule.extension.api.introspection.declaration.fluent.ParameterDeclaration;
+import org.mule.extension.api.introspection.property.PasswordModelProperty;
+import org.mule.module.extension.internal.introspection.ImmutablePasswordModelProperty;
+import org.mule.module.extension.internal.model.AbstractAnnotatedModelEnricher;
+import org.mule.module.extension.internal.model.property.DeclaringMemberModelProperty;
+
+import java.lang.reflect.Field;
+
+/**
+ * Enriches the {@link DeclarationDescriptor} with a model property which key is {@link PasswordModelProperty#KEY} and the value an instance of {@link PasswordModelProperty}.
+ *
+ * @since 4.0
+ */
+public final class PasswordModelEnricher extends AbstractAnnotatedModelEnricher
+{
+
+    @Override
+    public void enrich(DescribingContext describingContext)
+    {
+        DeclarationDescriptor descriptor = describingContext.getDeclarationDescriptor();
+        descriptor.getRootDeclaration().getDeclaration().getConfigurations().forEach(configurationDeclaration -> configurationDeclaration.getParameters().forEach(this::addModelPropertyIfRequired));
```

Comment:
this statement is just the same as the three below. Use a first order function to allow repetition. Repetition is the worst thing that can happen to an artist :)